docs/source/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -188,10 +188,16 @@ Here is a memo describing the rationale underlying the causal structure of this
      - TPP target
    * - x
      - crude birthweight in g shift between low and normal BMI women
-     - -160.44g (-196.65 to -124.23) fixed effect *or* -142.93g (-232.68 to -53.18) random effects
+     - -160.44g (-196.65 to -124.23) fixed effects *or* -142.93g (-232.68 to -53.18) random effects
```

Comment:
I thought it was "fixed effect" (singular) for meta-analysis, although I say Ali used the plural for the doc on the general method.  I will defer to the language she has selected after she double checks it in response to my prior comment

Reply:
ok